guides/v2.3/graphql/send-request.md

```diff
@@ -1,31 +1,16 @@
 ---
 group: graphql
-title: GraphQL request headers
+title: GraphQL requests
 ---
 
+In Magento 2.3.1, you must send all queries and mutations as HTTP POST requests. 
+
+## Request headers
+
 Magento accepts the following headers in a GraphQL request:
 
-<table>
-<tr>
-<th>Header name</th>
-<th>Description/Value</th>
-</tr>
-<tr>
-<td><code>Authorization</code></td>
-<td><p><code>Bearer <i>authorization-token</i></code></p>
-<p>An integration, admin, or customer token. See <a href="{{ page.baseurl }}/get-started/authentication/gs-authentication-token.html">Token-based authentication</a> for more information.</p>
-</td>
-</tr>
-<tr>
-<td><code>Store</code></td>
-<td><p>The store code on which to perform the request. The value can be one of the following:</p>
-<ul>
-<li><code>default</code></li>
-<li>A store code, which is defined when a store view is created.</li>
-<!--
-<li><p><code>all</code>. This value only applies to the CMS and Product modules. If this value is specified, the API call affects all the merchant's stores.</p></li>
--->
-</ul>
-</td>
-</tr>
-</table>
+Header name | Value | Description
+--- | --- | ---
+`Authorization` | `Bearer <authorization_token>` | A customer token. [Get customer authorization token]({{ page.baseurl }}/graphql/get-customer-authorization-token.html) describes how to generate the token.
+`Content-Type` | `application/json` | Required for all POST requests.
```

Comment:
Does this need to be defined here? Specifying this in your header doesn't give you anything special right? For graphql you need the customer or store token. Content-type and application/json are general to any HTTP request, which you specify in line 6.

Reply:
As of 2.3.2, queries can be sent via GET, in which case specifying the Content-Type can be optional. The in-browser GraphiQL tool we've been using gives you this for free. Other tooling might not.